dotnet/src/SemanticKernel.Abstractions/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:Microsoft.SemanticKernel.KernelFunctionMetadata.#ctor(System.String)</Target>
+    <Left>lib/netstandard2.0/Microsoft.SemanticKernel.Abstractions.dll</Left>
+    <Right>lib/netstandard2.0/Microsoft.SemanticKernel.Abstractions.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+</Suppressions>
```

Comment:
As a general rule we never want to introduce breaking changes.

We've done this early on in the 1.0 cycle and for changes related to experimental features but let's try to avoid this.

Reply:
I used @dmytrostruk's suggestion to eliminate this one.  Will see if something similar works with the other one.